scapy/contrib/avs.py

```diff
@@ -1,9 +1,21 @@
 #! /usr/bin/env python
 
-# http://trac.secdev.org/scapy/ticket/82
+# This file is part of Scapy
+# Scapy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# any later version.
+#
+# Scapy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scapy. If not, see <http://www.gnu.org/licenses/>.
 
 # scapy.contrib.description = AVS WLAN Monitor Header
-# scapy.contrib.status = loads
+# scapy.contrib.status = untested
```

Comment:
Not exactly, `untested` means that we don't expect it to load, and `loads` means that we don't except it to be fully functional. It its not related to unit tests. I agree that the semantic is not easy to understand.

If you really want to change theses values, I would suggest that you do a `load_contrib()` on the `contrib/*py` files to see if they work.